python/cugraph/cugraph/datasets/dataset.py

```diff
@@ -168,6 +168,9 @@ def get_edgelist(self, download=False):
         download : Boolean (default=False)
             Automatically download the dataset from the 'url' location within
             the YAML file.
+
+        create_using : module (default=cudf)
+            Specify between cudf and pandas to use when reading the dataset.
```

Comment:
This could be made more flexible if it accepted the function to call itself rather than a module.
```suggestion
        create_using : callable compatible with pandas.read_csv (default=cudf.read_csv)
            Object to call to read a CSV and return a pandas-compatible edgelist DataFrame.
```